src/Domain/CoordinateMaps/BulgedCube.cpp

```diff
@@ -157,6 +146,16 @@ boost::optional<std::array<double, 3>> BulgedCube::inverse(
   const double y_sq = square(physical_y);
   const double z_sq = square(physical_z);
   const double physical_r_squared = x_sq + y_sq + z_sq;
+  // There is no 'else' covering the case physical_r_squared==0.
```

Comment:
Should this say "physical_r_squared != 0" ? But also I think this line could be omitted, because IMO the following sentences are clear enough.

Reply:
I had previously meant to remove this line. Thanks for catching this!